examples/c/mass_spring_example.c

```diff
@@ -118,8 +126,9 @@ int main() {
                     ((ocp_qp_partial_condensing_args *)((ocp_qp_sparse_solver_args *)args)->pcond_args)->N2 = N2;
                     ((ocp_qp_hpmpc_args *)((ocp_qp_sparse_solver_args *)args)->solver_args)->max_iter = 30;
                     break;
+                #if ACADOS_WITH_QPDUNES
```

Comment:
It seems that you have this flag 2 times now, once in ocp_qp.c and once here. Can we not write it such that we only need it in the C interface?